pkg/defaults/policies/files/red_hat_images_signed.json

```diff
@@ -0,0 +1,99 @@
+{
+  "id": "c6fd7cb3-de34-4918-be4c-370006330357",
+  "name": "Red Hat images must be signed by a Red Hat release key",
+  "description": "Alert when Red Hat images are not digitally signed by Red Hat",
+  "rationale": "Red Hat images should be signed using Red Hat's product signing keys (https://access.redhat.com/security/team/key).",
+  "remediation": "Confirm the authenticity of the image by reaching out to Red Hat support.",
+  "disabled": true,
+  "categories": [
+    "Supply Chain Security"
+  ],
+  "lifecycleStages": [
+    "BUILD",
+    "DEPLOY"
```

Comment:
The lifecycle stages in this policy are build and deploy but the enforcement is only a build time enforcement. Is this intentional?

Given this is for redhat images only, I don't believe customers are building these in _their_ CI pipelines, so what is the use of having the build time enforcement?